package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hapi-pino",
-  "version": "11.0.0",
+  "version": "12.0.0",
```

Comment:
I'll bump this on release, please revert

Reply:
removed